include/mbedtls/ssl.h

```diff
@@ -185,6 +185,9 @@
 #define MBEDTLS_SSL_PRESET_DEFAULT              0
 #define MBEDTLS_SSL_PRESET_SUITEB               2
 
+#define MDEDTLS_SSL_SUPPRESS_CA_LIST_ENABLED     1
```

Comment:
Historical note: in the past we tended to favor a zero representation of the default as `memset(0)` was sort-of the official way to initialize structures. Now that `mbedtls_foo_init()` is generalized, I don't think there is a reason anymore to favor zero representation for the default value.